.github/workflows/system-tests-enterprise.yml

```diff
@@ -14,6 +14,9 @@
 #
 name: System Tests Enterprise
 
+env:
+  selectedbranch: 'development'
+
```

Comment:
better getting the content from a secret or pre-configured env
the reason is that we would want to configure the content externally and not change the code every time

Reply:
lets start with  define parameter , will change later if needed